### ISSUE TITLE
Add info about "key" column for StorIOSQLite annotations!

### ISSUE THREAD

**Issue by artem-zinnatullin** [pushtorefresh/storio#505]

**Comment by lectricas** [comment]
good idea!
Error:(10, 8) error: Class marked with StorIOSQLiteType annotation should have at least one KEY field marked with StorIOSQLiteColumn annotation
is really confusing

**Comment by artem-zinnatullin** [comment]
Isn't it kinda straightforward error message? :)

**Comment by lectricas** [comment]
Nope. I was like: What the hack is this key mean? A door key? Ctr+f on readme didn't help me. Then I figured it out by crawling the sample app. Consider adding "primary" word or something. You did it well on "StorIOSQLiteColumn" btw :D

**Comment by artem-zinnatullin** [comment]
Ahaha, got it :) Can you please make a PR then?
